File: 01 Introdução/1012 Área/1012.py
valores = input().split(' ')
A, B, C = valores
A = float(A)
B = float(B)
C = float(C)

areaTrianguloRetangulo = (A * C) / 2
areaCirculo = 3.14159 * C**2
areaTrapezio = (A+B)*C / 2
areaQuadrado = B**2
areaRetangulo = A * B

print(f"TRIANGULO: {areaTrianguloRetangulo:.3f}")
print(f"CIRCULO: {areaCirculo:.3f}")
print(f"TRAPEZIO: {areaTrapezio:.3f}")
print(f"QUADRADO: {areaQuadrado:.3f}")
print(f"RETANGULO: {areaRetangulo:.3f}")

# https://www.beecrowd.com.br/judge/pt/problems/view/1012
# Ranking 03349º, Time 0.003, Submiss Date 04/02/2023 09:08:10 (Python 3.9)

File: 01 Introdução/1012 Área/1012.java
import java.util.Scanner;

class Main {
  public static void main(String args[]) {
    Scanner sc = new Scanner(System.in);

    float A = sc.nextFloat();
    float B = sc.nextFloat();
    float C = sc.nextFloat();

    System.out.printf("TRIANGULO: %.3f\n", (A * C) / 2);
    System.out.printf("CIRCULO: %.3f\n", 3.14159 * C * C);
    System.out.printf("TRAPEZIO: %.3f\n", (A + B) * C / 2);
    System.out.printf("QUADRADO: %.3f\n", B * B);
    System.out.printf("RETANGULO: %.3f\n", A * B);

    sc.close();

  }
}

// https://www.beecrowd.com.br/judge/pt/problems/view/1012
// Ranking 18032º, Time 0.296, Submiss Date 12/02/2023 22:49:14 (Java 14)